sbt-typesafe-conductr-tester/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+lazy val root = Project("plugins", file(".")).dependsOn(plugin)
+lazy val plugin = file("../").getCanonicalFile.toURI
```

Comment:
Re-instated the tester - @ktoso it is there for the ease of experimenting during development. Josh showed me this convention a while back.